test/membrane/error.spec.js

```diff
@@ -77,7 +77,7 @@ it('[red] unhandled promise rejections with non-error objects and red listener',
     const env = createVirtualEnvironment(window, window, {
         endowments: Object.getOwnPropertyDescriptors({ done }),
     });
-
+    debugger;
     env.evaluate(`
```

Comment:
☝️ Test at the debugger break point with `yarn run karma:debug` then open Safari TP and go to the URL too